### PR TITLE
Minor corrections to regexes section

### LINKED ISSUE #2873: Character classes also capture
## The problem

[Predefined character classes](https://docs.perl6.org/language/regexes#Predefined_character_classes) not only match, they also capture, putting a list of matched characters into a list

## Suggestions

Please mention that in the documentation

### PR DESCRIPTION
**Pull request by threadless-screw**
Also fixes #2873

### CODE REVIEW

File: doc/Language/regexes.pod6 [hunk 1]
```diff
@@ -261,9 +261,16 @@ Examples of word characters:
...
-so called zero-width assertions, which do not really match a
-character.
+The predefined character classes in the leftmost column are all of the form
+C«<name>», a hint to the fact that they are implemented as built-in L<named
+regexes|language/regexes#Subrules>. As such they are subject to the usual
+capturing semantics. This means, inter alia, that these classes do not only
+match, but also capture, installing a correspondingly named L<named
+capture|/language/regexes#Named_captures> in the resulting L<Match
+object|/type/Match>.
+
```
Comment (JJ) [inline doc/Language/regexes.pod6]: Please mention that preceding them with a . prevents capture.

File: doc/Language/regexes.pod6 [hunk 2]
```diff
@@ -44,7 +44,7 @@ operators:
     $match = m/.+/; say $match; say $match.^name; # OUTPUT: «｢abc｣␤Match␤»
     $match =  /.+/; say $match; say $match.^name; # OUTPUT: «/.+/␤Regex␤»
 
-Whitespace in literal regexes is ignored insofar as the L<:sigspace>
+Whitespace in literal regexes is ignored insofar as the L<:sigspace
```
Comment (JJ) [inline doc/Language/regexes.pod6]: I think that it's actually a `C<` what's missing here. Also try to fit in in the same line.

Comment (threadless-screw) [reply doc/Language/regexes.pod6]: Thanks - you're right.

File: doc/Language/regexes.pod6 (hunk 1 above)
Comment (threadless-screw) [reply doc/Language/regexes.pod6]: Done.

**Pull request merged**